### PR TITLE
Fix get_team_name

### PR DESCRIPTION
**Pull request by cgundy**
Found a small bug related to how team names are parsed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ ic
 sdk
 ```
 
-To unlock a repository, you should create a [Pull Request](https://github.com/dfinity/repositories-open-to-contributions/compare) where you add the name in alphabetical order.
+To unlock a repository, you should create a [Pull Request](https://github.com/dfinity/repositories-open-to-contributions/compare) where you add the name in alphabetical order. Be sure to commit this directly to the repository, a PR from a forked repository will not work.
```
Comment (levshaATdfinity) [inline README.md:13]: Why it won't work and why can't we make it working?
Also the change is not related to the main fix of this PR I believe.

File: .github/custom_python_actions/compliance_checks.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def get_team_name(code_owners: str, org_name: str) -> str:
     if len(team_handle) > 1:
         raise Exception("Only one team can be listed for repo-level codeowners.")
     codeowner_team = team_handle[0].lower()
-    team_name = codeowner_team.strip(f"@{org_name.lower()}").strip("/")
+    team_name = codeowner_team.replace(f"@{org_name.lower()}/", "")
```
Comment (levshaATdfinity) [inline .github/custom_python_actions/compliance_checks.py:41]: Add test for this?

File: README.md (hunk 1 above)
Comment (cgundy) [reply README.md:13]: I know - I snuck it in. This is the issue related to not being able to pass in secrets from a forked repo. https://github.com/openmainframeproject/feilong/issues/367

File: .github/custom_python_actions/compliance_checks.py (hunk 2 above)
Comment (cgundy) [reply .github/custom_python_actions/compliance_checks.py:41]: We actually already have tests for this here: https://github.com/dfinity/repositories-open-to-contributions/blob/355ccd7a907290aa24434737ad3714b7ba2294c5/.github/tests/test_compliance_checks.py#L67

It was a very bizarre error - it seemed to only be an issue with the letter "t"? If we want to be thorough, I can add another test for that specific case, but because the tests still passed after this change I feel confident with it.

File: README.md (hunk 1 above)
Comment (cgundy) [reply README.md:13]: I think this is intentional for security reasons.

File: .github/custom_python_actions/compliance_checks.py (hunk 2 above)
Comment (levshaATdfinity) [reply .github/custom_python_actions/compliance_checks.py:41]: Just ad a case that would fail on the current code and pass with this change?

File: README.md (hunk 1 above)
Comment (levshaATdfinity) [reply README.md:13]: Yes, not sending secrets to pipelines running on forked repositories is intentional. We may still make it working probably as `GITHUB_TOKEN` can be still accessible from PRs on forked repositories, but that should be separate task.
Anyway the comment still looks confusing to me and might be misleading, because "commit this directly to the repository" might be misinterpreted as "commit directly to the master"

Comment (cgundy) [reply README.md:13]: Sure, the problem is that we can't use GITHUB_TOKEN, but need to use a custom PAT to access org membership. But sure, I'll update the comment. 👍

**Review by levshaATdfinity — APPROVED** [review]
